### ISSUE TITLE
Support mixing remote and local folders

### ISSUE THREAD

**Issue by Christilut** [fabiospampinato/vscode-projects-plus#59]
Thanks for this extension! The only problem I have is that it seems to install locally and on the remote (WSL in my case) separately. I switch between projects a lot and some can't be run in WSL so I'm forced to use Windows for those.

Is there any way to combine the settings?

**Comment by fabiospampinato** [comment]
I've no idea, I have no use for WSL nor remote containers, PRs welcome.

**Comment by Christilut** [comment]
Looks like this API is also missing :/ I put out a question on the VSCode repo, I'll report back if there is an update

**Comment by flammehawk** [comment]
Well one option for Containers at least would be to mount the Projectsjson file into the Container.

**Comment by jmarandet** [comment]
Hey, did you made any progress on this feature ?

I think it's too bad this plugin doesn't support Remote WSL, since this is a real breaking change
In my point of view, Project+ is really the best module for project management in VSCode, the UI experience is just amazing, and you did a really great job

But today, WSL for VSCode is a real shift ; to taste it is to adopt it

Despite, I am sadly forced to use Project Manager (https://marketplace.visualstudio.com/items?itemName=alefragnani.project-manager) which is sooooo boring...  with a hundred of paths to register

**Comment by jaycapps** [comment]
Agreed.  This is my preferred project management solution and support for remote WSL is becoming a real need, at least for me.
